src/env.cc

```diff
@@ -231,6 +254,23 @@ void Environment::Start(const std::vector<std::string>& args,
   HandleScope handle_scope(isolate());
   Context::Scope context_scope(context());
 
+  if (*TRACE_EVENT_API_GET_CATEGORY_GROUP_ENABLED(
+      TRACING_CATEGORY_NODE1(environment))) {
```

Comment:
Nit: `!= 0` for clarity? IIUC

Reply:
:thinking: makes sence but we have https://github.com/nodejs/node/blob/561e30d9ef581e86b36318fe22ebd1e82ab88754/src/node_contextify.cc#L661-L662 and https://github.com/nodejs/node/blob/561e30d9ef581e86b36318fe22ebd1e82ab88754/src/node_file.cc#L92-L93
?

Reply:
doesn't hurt to have the `!= 0` ... we can refactor them all out later if needed.